### ISSUE TITLE
Questions by using pre-optimized models

### ISSUE THREAD

**Issue by starcxl** [facebookresearch/banmo#53]
Dear GengShan,

Hi! Many thanks for your awesome job! I am trying to use your pre-optimized models for evaluation. 

`bash scripts/render_mgpu.sh 0 human-cap logdir/human-cap/human-cap.pth "0" 64
`

However, the results seem very strange as below, including the viewing directions and model motions. 

![human-cap-{0}-all](https://user-images.githubusercontent.com/131245463/233018674-43dbfd4d-5536-434b-80ac-44e392bb06d5.gif)
Could you please tell me what's happening to these questions?

Thanks!

**Comment by gengshan-y** [comment]
Hi, could you try [this](https://www.dropbox.com/s/71zbu72z4tv1mor/opts-human.log?dl=0) file? (need to save it as opts.log in the same directory as pre-trained models). I realized the one provided in the pre-trained model link is for quadruped animals.

This runs without problem for me.

**Comment by gengshan-y** [comment]
It should work without the init-cam folder. That not being used at inference time.

Was there a warning when executing the code with the following message?
`!!!deleting video specific dicts due to size mismatch!!!`

It would be best if you attach the a log from the console. Here's mine:
[log.txt](https://github.com/facebookresearch/banmo/files/11287743/log.txt)

**Comment by starcxl** [comment]
Many thanks for your reply! Yes, there is a warning as you mentioned.  And I got the same warning and the same strange results when running the cat example. I attach my log: [log.txt](https://github.com/facebookresearch/banmo/files/11293322/log.txt)
!!!deleting video specific dicts due to size mismatch!!!



I check the warning code [here](https://github.com/facebookresearch/banmo/blob/b9dab8084907197d7a7ede45f6fc7202736bf9be/nnutils/train_utils.py#L317). For my situation, the states['near_far'].shape[0] is 583, while the self.model.near_far.shape[0] is 582. 

I don't know what causes the mismatch. Could you please tell me the potential reasons?

**Comment by gengshan-y** [comment]
![image](https://user-images.githubusercontent.com/13134872/233651171-46697856-1d69-4f69-8a79-195e0debfdeb.png)

This suggests you are missing an image in `database/DAVIS/JPEGImages/Full-Resolution/human-cap006/`, and below is my folder (adult-500 is the old name of human-cap006)

<img width="731" alt="image" src="https://user-images.githubusercontent.com/13134872/233651882-dff58c9b-4d63-48ef-bc15-c6219051cab9.png">

**Comment by starcxl** [comment]
Oh! It works now if I replace human-cap006 datasets with the corresponding datasets you provided. 

For the absent image issue, I retried preprocessed command below, but still, it extracts 00000-00037.jpg from human-cap_6.MOV, with 00038.jpg missing. (using ffmeng 3.4.8, pytorch 1.7.1, cuda 11.0)
[bash preprocess/preprocess.sh human-cap_6 .MOV y 10](url)

I am confused about what causes this difference. Could you please retry the command above to see if 00038.jpg could be extracted normally?

**Comment by gengshan-y** [comment]
Can confirm it works for me. 

In the preprocessing script, we first use ffmpeg to extract frames, and then the frames are moved to tmp/$seqname/images.
<img width="764" alt="image" src="https://user-images.githubusercontent.com/13134872/233806682-b25d0966-6685-4024-9e9c-f65e58816ac1.png">
Then preprocess/mask.py reads images and runs segmentation. The frames with good detections are saved to database/DAVIS/JPEGImages/...

Attaching the log of running mask.py below:
[log.txt](https://github.com/facebookresearch/banmo/files/11302279/log.txt)


Can you confirm ffmpeg or mask.py are producing expected outputs?
